spec/lib/active_campaign/client_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+RSpec.describe ActiveCampaign::Client, :vcr do
+  describe '.new' do
+    it 'can be used to intialize a client object that can make requests' do
+      WebMock.stub_request(:get, 'http://fake.com/').with(body: '{}')
+      described_class.new.get('http://fake.com/')
```

Comment:
That's a good smoke test, perhaps we could actually add an expectation as well? Just to make sure that we are not just relying on that an error is not raised? I also think the webmock stuff isnt' working.

Reply:
You're right -- great catch. The spec wasn't doing exactly what I thought it was. I think I've fixed the issue and also now explicitly check that the request is made. How does it look?